test/junit/scala/collection/WithFilterTest.scala

```diff
@@ -13,7 +14,7 @@ class WithFilterTest {
         x <- List(1, 2, 3)
         if x % 2 == 0
       } yield x + 1
-    val xs1t: List[Int] = xs1
+    @unused val xs1t: List[Int] = xs1
```

Comment:
Common idiom for type test. Sometimes expressed as ascription `xs1: List[Int]` without assignment to var.